docker/Dockerfile

```diff
@@ -52,11 +52,19 @@ RUN apt-get update &&apt-get install -y \
     libsgx-ra-uefi
 
 
-# Build App from source
+#  Install jsoncpp && Build App from source
 WORKDIR /home
-RUN git clone https://github.com/intel/ehsm.git \
-   && cd ehsm \
-   && make
+RUN git clone https://github.com/microsoft/vcpkg.git \
+    && git clone -b enroll_M0 https://github.com/Liang1XZhao/ehsm.git \
```

Comment:
https://github.com/Liang1XZhao/ehsm.git ?

Reply:
it will be fix